src/Pillar-Tests-ExporterCore.package/PRDocumentListTransformerTest.class/instance/createTemplateFile.st

```diff
@@ -0,0 +1,7 @@
+private
+createTemplateFile
+	| dir templateFile |
```

Comment:
I do not understand. We crate the file if it does not exist? Why not doing everything in memory directly?